modules/swagger-codegen/src/main/resources/qt5cpp/helpers-header.mustache

```diff
@@ -11,11 +11,10 @@ namespace {{this}} {
 {{/cppNamespaceDeclarations}}
 
     void setValue(void* value, QJsonValue obj, QString type, QString complexType);
-    void toJsonArray(QList<void*>* value, QJsonObject* output, QString innerName, QString innerType);
-    void toJsonValue(QString name, void* value, QJsonObject* output, QString type);
-    void toJsonMap(QMap<QString, void*>* value, QJsonObject* output, QString innerName, QString innerType);
+    void toJsonArray(QList<void*> value, QJsonObject& output, QString innerName, QString innerType);
+    void toJsonValue(QString name, void* value, QJsonObject& output, QString type);
+    void toJsonMap(QMap<QString, void*> value, QJsonObject& output, QString innerName, QString innerType);
     bool isCompatibleJsonValue(QString type);
-    QString stringValue(QString* value);
```

Comment:
QString stringValue(QString* value) is used in the code, strangely it compiles. But when I am running the SW the server throws error saying ?true in Query Path because for whatever reason Qt has used the bool version of the function overload. Very weird.